test/Dialect/Basicpy/ops_invalid.mlir

```diff
@@ -0,0 +1,49 @@
+// RUN: npcomp-opt -split-input-file -verify-diagnostics %s
```

Comment:
nit: ops-invalid.mlir ? (don't know which way the wavefunction has collapsed in this part of the repo)